diff --git a/node_modules/@electron/remote/index.d.ts b/node_modules/@electron/remote/index.d.ts
index 26836fb..b4d0d32 100644
--- a/node_modules/@electron/remote/index.d.ts
+++ b/node_modules/@electron/remote/index.d.ts
@@ -9,6 +9,7 @@ export {
   DownloadItem,
   IncomingMessage,
   MessagePortMain,
+  SafeStorage,
   ServiceWorkers,
   TouchBarButton,
   TouchBarColorPicker,
@@ -51,6 +52,7 @@ export var screen: Electron.Screen;
 export var session: typeof Electron.session;
 export var ShareMenu: typeof Electron.ShareMenu;
 export var shell: Electron.Shell;
+export var safeStorage: Electron.SafeStorage;
 export var systemPreferences: Electron.SystemPreferences;
 export var TouchBar: typeof Electron.TouchBar;
 export var Tray: typeof Electron.Tray;
