diff --git a/node_modules/@types/node-microphone/index.d.ts b/node_modules/@types/node-microphone/index.d.ts
index d376d0c..5faa0c8 100644
--- a/node_modules/@types/node-microphone/index.d.ts
+++ b/node_modules/@types/node-microphone/index.d.ts
@@ -8,6 +8,7 @@ interface MicrophoneOptions {
     channels?: 1 | 2 | undefined;
     device?: "hw:0,0" | "plughw:1,0" | "default" | undefined;
     additionalParameters?: any;
+    binary?: string;
 }
 
 declare class Microphone {
