diff --git a/node_modules/node-microphone/index.js b/node_modules/node-microphone/index.js
index 0dcb2cf..8d6218e 100644
--- a/node_modules/node-microphone/index.js
+++ b/node_modules/node-microphone/index.js
@@ -10,6 +10,7 @@ class Microphone extends EventEmitter {
         this.ps = null;
 
         options = options || {};
+        this.binary = options.binary;
         this.endian = options.endian || 'little';
         this.bitwidth = options.bitwidth || '16';
         this.encoding = options.encoding || 'signed-integer';
@@ -70,7 +71,7 @@ class Microphone extends EventEmitter {
                         this.additionalParameters
                     );
                 }
-                this.ps = spawn('sox', audioOptions);
+                this.ps = spawn(this.binary || 'sox', audioOptions);
             } else if (isMac) {
                 audioOptions = [
                     '-q',
